services/logs2webhook/Dockerfile

```
@@ -0,0 +1,34 @@
+ARG LAGOON_GIT_BRANCH
+ARG IMAGE_REPO
+ARG UPSTREAM_REPO
+ARG UPSTREAM_TAG
+# STAGE 1: Loading Image lagoon-node-packages-builder which contains node packages shared by all Node Services
+FROM ${IMAGE_REPO:-lagoon}/yarn-workspace-builder as yarn-workspace-builder
+
+# STAGE 2: specific service Image
+FROM ${UPSTREAM_REPO:-uselagoon}/node-10:${UPSTREAM_TAG:-latest}
```

```suggestion
FROM ${UPSTREAM_REPO:-uselagoon}/node-16:${UPSTREAM_TAG:-latest}
```
node-16 as per the others